rule/return-argument-limit.go

```diff
@@ -0,0 +1,65 @@
+package rule
+
+import (
+	"fmt"
+	"go/ast"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// ReturnArgumentsLimitRule lints given else constructs.
+type ReturnArgumentsLimitRule struct{}
```

Comment:
Formally, functions do not return _arguments_ but _results_ (https://golang.org/ref/spec#Signature). 
What do you think about renaming the rule to something like `FunctionResultsLimit` ?